io/eolearn/io/sentinelhub_process.py

```diff
@@ -301,7 +283,7 @@ def _create_response_objects(self):
 
         return responses
 
-    def _get_timestamp(self, time_interval, bbox):
+    def _get_timestamp(self, time_interval: Optional[RawTimeIntervalType], bbox: BBox) -> List[dt.datetime]:
```

Comment:
Optional[RawTimeIntervalType], bbox: BBox) -> List[RawTimeIntervalType] ?

Reply:
what we get out are timestamps, and not intervals